### PR TITLE
Logging fixes

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

File: akanda/rug/main.py [hunk 1]
```diff
@@ -104,18 +130,6 @@ def register_and_load_opts(argv):
         cfg.StrOpt('rpc-exchange',
                    default='l3_agent_fanout',
                    help='name of the exchange where we receive RPC calls'),
-
-        # These should stay as cli opts
-        cfg.BoolOpt('debug',
```
Comment (ryanpetrello) [inline akanda/rug/main.py:109]: Will the removal of these cause our chef scripts to fail somewhere?

Comment (dhellmann) [inline akanda/rug/main.py:109]: The debug and verbose options are defined in the oslo logging code, so they should still work.

Comment (ryanpetrello) [inline akanda/rug/main.py:109]: Makes sense :)

**Comment by ryanpetrello** [comment]
LTGM

**Pull request merged**